<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerWidget</class>
 <widget class="QWidget" name="ControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="mReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mQuit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="mScene1">
       <property name="text">
        <string>Scene1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mScene1Int"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="mRotateAll">
       <property name="text">
        <string>RotateAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fRotateEdit">
       <property name="text">
        <string>rect*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
